components/Buttons/src/MDCFloatingButton.m

```diff
@@ -71,55 +112,171 @@ - (instancetype)initWithFrame:(CGRect)frame shape:(MDCFloatingButtonShape)shape
 - (instancetype)initWithCoder:(NSCoder *)aDecoder {
   self = [super initWithCoder:aDecoder];
   if (self) {
+    [self commonMDCFloatingButtonInit];
```

Comment:
I think it makes sense to leave the `commonInit` call here in case the button is unarchived from a storyboard/xib. In that case we explicitly initialize defaults. Although properties are assigned zeroed memory, I think it's safer to use explicit assignments so any future changes to default values can be easily identified with a string search.

![screen shot 2017-11-29 at 12 56 01 pm](https://user-images.githubusercontent.com/1753199/33390682-a6dcbb08-d504-11e7-8930-8dde48275642.png)